src/test/java/org/nuxeo/binary/metadata/test/TestBinaryMetadataService.java

```diff
@@ -90,6 +95,15 @@
         inputPSDMetadata.put("EXIF:ImageHeight", "200");
         inputPSDMetadata.put("EXIF:Software", "Nuxeo");
         inputPSDMetadata.put("IPTC:Keywords", new String[] {"keyword1", "keyword2"});
+        inputPSDMetadata.put("IPTC:Keywords", new String[] {"keyword1", "keyword2"});
+        try {
+            Date date = new SimpleDateFormat("yyyy:MM:dd HH:mm:ss").parse("2018:06:15 00:00:00");
+            Calendar calendar = Calendar.getInstance();
+            calendar.setTime(date);
+            inputPSDMetadata.put("EXIF:DateTimeOriginal", calendar);
+        } catch (ParseException e) {
+            fail(e.getMessage());
```

Comment:
Why this form of catch/fail? Why not simply let the exception bubble up to the test framework, if it normally isn't produced?